src/main/api-utilities/SinglePropertyVerifier.js

```diff
@@ -0,0 +1,64 @@
+export class SinglePropertyVerifier {
+    constructor (scenario) {
+        this.scenario = scenario;
+    }
+
+    check () {}
```

Comment:
I believe we don't need this empty function here, if you have it in the other classes that extends SinglePropertyVerifier